android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java

```diff
@@ -946,20 +946,20 @@ public void onNetworkQualityLevelChanged(@NonNull LocalParticipant localParticip
                 String unstableConnection = pluginOptions.getString(UNSTABLE_CONNECTION_TEXT, "Unstable connection");
                 String badConnection = pluginOptions.getString(UNSTABLE_CONNECTION_TEXT, "Bad connection");
 
-                if (networkQualityLevel == NetworkQualityLevel.NETWORK_QUALITY_LEVEL_FIVE || networkQualityLevel == NetworkQualityLevel.NETWORK_QUALITY_LEVEL_FOUR) {
-                    networkQuality.setVisibility(View.INVISIBLE);
-                } else if (networkQualityLevel == NetworkQualityLevel.NETWORK_QUALITY_LEVEL_THREE) {
-                    int color = Color.parseColor("#FFEB3B");
+                if (networkQualityLevel == NetworkQualityLevel.NETWORK_QUALITY_LEVEL_ONE) {
```

Comment:
There is also a NETWORK_QUALITY_LEVEL_ZERO shouldn't it be considered a bad connection? On the other hand, the reconnecting message will show up in the middle of the screen.

Reply:
oh, good catch!!